QuantConnect.BybitBrokerage/BybitBrokerage.Brokerage.cs

```diff
@@ -134,21 +134,31 @@ public override bool PlaceOrder(Order order)
             return false;
         }
 
-        var submitted = false;
-
         _messageHandler.WithLockedStream(() =>
         {
-            var result = ApiClient.Trade.PlaceOrder(GetBybitProductCategory(order.Symbol), order,
-                useMargin: _algorithm.BrokerageModel.AccountType == AccountType.Margin);
+            var result = default(Models.Requests.BybitUpdateOrderResponse);
+            try
+            {
+                result = ApiClient.Trade.PlaceOrder(GetBybitProductCategory(order.Symbol), order,
+                    useMargin: _algorithm.BrokerageModel.AccountType == AccountType.Margin);
+            }
+            catch (Exception ex)
+            {
```

Comment:
Is the `OrderId` available in the exception? If so, we should call
`order.BrokerId.Add(result.OrderId);`
before the order event, so users can use the `OrderId` to ask the brokerage if they have to.